src/concern/mod.rs

```diff
@@ -29,6 +29,29 @@ pub struct ReadConcern {
     pub level: ReadConcernLevel,
 }
 
+impl ReadConcern {
```

Comment:
I added these utility constants (and the equivalent for `WriteConcern`) because it's _way_ more readable to construct a collection that uses majority read and write concern using them than having to nest construction of the concern structs (e.g. in `ClientEncryption::new`).

Reply:
nice, yeah the construction of these can be pretty verbose